tests/ignite/metrics/test_mean_absolute_error.py

```diff
@@ -42,19 +42,15 @@ def get_test_cases():
 
         test_cases = [
             (torch.randint(0, 10, size=(100, 1)), torch.randint(0, 10, size=(100, 1)), 1),
-            (torch.randint(-10, 10, size=(100, 1)), torch.randint(-10, 10, size=(100, 1)), 1),
-            (torch.randint(0, 20, size=(100, 5)), torch.randint(0, 20, size=(100, 5)), 1),
-            (torch.randint(-20, 20, size=(100, 5)), torch.randint(-20, 20, size=(100, 5)), 1),
```

Comment:
Why did we remove those test cases?

Reply:
You can check some of our previous PRs we agreed on reducing the computational load, to do that we changed the range of the for loop to 5 instead of 10 and less test cases